bin/pam_ball_trajectories.py

```diff
@@ -0,0 +1,209 @@
+        print("\nhdf5 trajectories file: {}".format(hdf5_path))
+        if group_name:
+            _info_group(rbt, group_name)
+        else:
+            _info_whole_file(rbt)
+
+
+def _add_json(hdf5_path: pathlib.Path, group_name: str, sampling: int):
+    logging.info("recording trajectories in {}".format(hdf5_path))
+    with bt.RecordedBallTrajectories(path=hdf5_path) as rbt:
+        if group_name in rbt.get_groups():
+            raise ValueError("group {} already present in the file")
+        nb_added = rbt.add_json_trajectories(group_name, pathlib.Path.cwd(), sampling)
+    logging.info("added {} trajectories".format(nb_added))
+
+
+def _add_tennicam(hdf5_path: pathlib.Path, group_name: str):
+    logging.info("recording trajectories in {}".format(hdf5_path))
+    with bt.RecordedBallTrajectories(path=hdf5_path) as rbt:
+        if group_name in rbt.get_groups():
+            raise ValueError("group {} already present in the file")
+    nb_added = rbt.add_tennicam_trajectories(group_name, pathlib.Path.cwd())
+    logging.info("added {} trajectories".format(nb_added))
+
+
+def _rm_group(hdf5_path: pathlib.Path, group_name: str):
+    with bt.RecordedBallTrajectories(path=hdf5_path) as rbt:
+        rbt.rm_group(group_name)
+
+
+def _translate(hdf5_path: pathlib.Path, group_name: str, coords: typing.List[float]):
+    coords = np.array(coords, np.float32)
+    with bt.RecordedBallTrajectories(path=hdf5_path) as rbt:
+        for index in rbt.get_indexes(group_name):
+            stamps, trajectory = rbt.get_stamped_trajectory(
+                group_name, index, direct=True
+            )
+            trajectory += coords
+            rbt.overwrite(group_name, index, (stamps, trajectory))
+
+
+def run():
+
+    parser = argparse.ArgumentParser()
+
+    # the executable will use by default the hdf5 managed by
+    # pam_configuration, but user has the option to point to
+    # another file.
+    parser.add_argument(
+        "--path",
+        type=str,
+        required=False,
+        help="path to the hdf5 file encapsulating the ball trajectories",
+    )
+
+    # 5 commands supported: info, add-json, add-tennicam,
+    # rm and translate.
+    subparser = parser.add_subparsers(dest="command")
+
+    # for displaying info about the hdf5 file
+    info = subparser.add_parser(
+        "info",
+        help="print information about the whole file, or a given group in the file.",
+    )
+    info.add_argument(
+        "--group", type=str, required=False, help="the group of trajectories"
+    )
+
+    # for adding the json files of the current folder
+    # to a hdf5 trajectory file
+    add_json = subparser.add_parser(
+        "add-json",
+        help="for saving in a new group all json trajectories present in the current directory",
+    )
+    add_json.add_argument(
+        "--group", type=str, required=True, help="the group of trajectories"
+    )
+    add_json.add_argument(
+        "--sampling_rate_us",
```

Comment:
I think it is more conventional to use dashes instead of underscores in parameter names.

Reply:
+1